docs/runtime/file-types.mdx

```diff
@@ -84,6 +86,29 @@ export default {
 
 </CodeGroup>
 
+### `jsonc`
```

Comment:
upon fixing that issue, I realised this file-types.mdx file was also outdated so i fix it too